geniza/corpus/views.py

```diff
@@ -37,17 +40,12 @@ def get_form_kwargs(self):
         # use GET instead of default POST/PUT for form data
         form_data = self.request.GET.copy()
 
-        # always use relevance sort for keyword search;
-        # otherwise use default (sort by name)
-        if form_data.get("q", None):
+        # sort by chosen sort
+        if "sort" in form_data:
+            form_data["sort"] = form_data.get("sort", None)
+        # sort by relavance if query text exists and no sort chosen
+        elif form_data.get("q", None):
             form_data["sort"] = "relevance"
-        # sorting TODO
-        # else:
-        # form_data['sort'] = self.initial['sort']
-
-        # use initial values as defaults
-        # for key, val in self.initial.items():
-        # form_data.setdefault(key, val)
 
```

Comment:
the default initial sort isn't getting set; I had to look at the django docs again to be reminded of this:
> These values are only displayed for unbound forms, and they’re not used as fallback values if a particular value isn’t provided.

So if we're setting form data and binding the form, we need to pass the initial sort value into our form data.